### PR TITLE
Adding code of conduct + contributing readme docs to the development branch

### PR DESCRIPTION
**Pull request by shriyash**
# Fixes # 

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
-

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=47) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=47) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/dx-automator?pullRequest=47) it.</sub>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+We welcome direct contributions to the sendgrid-csharp code base. Thank you!
+
+### Development Environment ###
+
+#### Install and Run Locally ####
+
+##### Prerequisites #####
+
+- .NET 4.5.2+
```
Comment (thinkingserious) [inline CONTRIBUTING.md]: Please update the content in this file to be relevant to this repo. Thanks!

**Comment by thinkingserious** [comment]
Hello @shriyash,

Please resolve the conflicting files.

Also, please remove the CONTRIBUTING.md file as it does not pertain to this repo. This is not a C# repo.

Thanks!

With Best Regards,

Elmer

**Comment by thinkingserious** [comment]
Hello @shriyash,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**